core/src/main/java/org/apache/calcite/sql/dialect/MssqlSqlDialect.java

```diff
@@ -114,8 +115,25 @@ public MssqlSqlDialect(Context context) {
       case TRIM:
         unparseTrim(writer, call, leftPrec, rightPrec);
         break;
+      case CONCAT:
+        final SqlWriter.Frame concatFrame = writer.startFunCall("CONCAT");
+        for (SqlNode operand : call.getOperandList()) {
+          writer.sep(",");
```

Comment:
Is there a better way to implement the ','

Reply:
writing in last will add an extra comma in the end